packages/config/src/ProfileInfo.ts

```diff
@@ -17,6 +17,7 @@ import { IProfAttrs } from "./doc/IProfAttrs";
 import { IProfArgAttrs } from "./doc/IProfArgAttrs";
 import { IProfLoc, ProfLocType } from "./doc/IProfLoc";
 import { IProfMergedArg } from "./doc/IProfMergedArg";
+import { IProfOpts } from "./doc/IProfOpts";
 
 // for team config functions
 import { Config } from "./config";
```

Comment:
Failing to build because of file pathing error. `./config` should be `./Config`

Reply:
Thanks for finding this. Something else always seemed more urgent than looking into pipeline errors, since it built and tested ok on my Windows laptop.